packages/preview-middleware-client/package.json

```diff
@@ -26,9 +26,12 @@
     "engines": {
         "node": ">=18.x"
     },
-    "dependencies": {},
+    "dependencies": {
+        "semver": "7.6.3"
```

Comment:
What particular issue did you want to solve with this dependency? So far we don't have any external dependencies and I don't think we should change that without a good reason. What I can see that it replace two if statements, does not look like good enough reason for adding this, but maybe there is something else I'm not aware of.

Reply:
Fair point. Current code version from main completely ignores the patch version so I thought using semver lt is easier than to compare all those version parts in various if statements.
Nevertheless semver might be a bit too much as we don't expect to get any ranges from the UI5 version fetch. I just recently discovered it and now maybe want to use it too often 😄 Don't know. Maybe you are right and it's not worth introducing the first dependency here.